ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/validation/ModelConformanceValidatorSpec.scala

```diff
@@ -375,6 +375,20 @@ class ModelConformanceValidatorSpec
         .build()
       step shouldBe StepStop(expectedEntry)
     }
+
+    "accept a transaction in case an input contract ID is non-existent" in {
+      val step = defaultValidator
+        .validateCausalMonotonicity(
+          aTransactionEntry,
```

Comment:
Not sure if we can easily improve anything, but here the naming is quite "obfuscating". There's no easy way of telling if this transaction has a contract or not which is not included in the commit context., you have either check it every time you write/read a test or just memorize this abstrac test namings.

Reply:
I've added a comment to note that there are no contracts present in the commit context.